packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -396,7 +396,19 @@ class _DiffViewState extends State<DiffView> with AutoDisposeMixin {
               axis: Axis.vertical,
               children: [
                 _buildTreemap(),
-                CodeSizeDiffTable(rootNode: diffRoot),
+                Row(
+                  children: [
+                    Flexible(
```

Comment:
Why Flexible instead of expanded? In general I think Expanded is a safer default as you won't end up with UI that doesn't fit the full width of the row.

Reply:
Because each item in the row needs to have a flex value of one